### ISSUE TITLE
[Screen Reader - Intelligent Support - IRIS]: The label is not announced by the screen reader when focus lands on “C.6908323246 - UI Submission workflow button”.

### ISSUE THREAD

**Issue by pgopinadhu** [microsoft/BotFramework-WebChat#4554]
**PWD Impact:**
Users who are dependent on-screen readers would be left confused if label is not announced simultaneously with the button.

**Repro Steps:**
Step 1: Open the above URL in Browser
Step 2: Navigate to IRIS bot icon and invoke it. IRIS chat bot is displayed
Step 4: Type start over-> create Invoice
Step 5: IRIS Bot replies with sales order number
Step 6: Enable Narrator and navigate to message and verify if the screen reader announces the label.

**Actual Result:**
When we navigate to message, screen reader is not announcing the label.

**Expected Result:**
Screen reader should announce the label ““Please select **engagement/sales order number** from below list or type valid **engagement/sales order number**” simultaneously with button “C.6908323246 - UI Submission workflow button”, when we navigate to using tab key to the button

**Observation:**  NVDA also does not announce the label

PFA<MAS3.3.2 - label is not announced_NVDA, MAS3.3.2 - label is not announced>

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/sites/accessibility/_layouts/15/Doc.aspx?sourcedoc=%7b3752B546-30D8-44D7-8221-6FC32EEB3AC8%7d&file=MAS%203.3.2%20%E2%80%93%20Labels%20or%20Instructions.docx&action=default&mobileredirect=true

**Test Environment:**
**OS:** Windows 11 Enterprise version 21H2
**Browser:** New Edge Version 106.0.1370.42 (Official build) (64-bit)
**URL:** https://servicesuat.microsoft.com/#/

**Comment by pgopinadhu** [comment]
#A11yMAS; #A11ySev2; #HCL; #Accessibility; #Win11-Edge(Chromium); #MAS3.3.2; #BotFrameworkWebChat-Jul22

**Comment by compulim** [comment]
@pgopinadhu I couldn't find the Iris bot in the header. The Resource360 Dashboard appears empty to me.

Can you verify if the Iris bot is running?

![image](https://user-images.githubusercontent.com/1622400/211652823-b1f798d1-e3e6-4c92-ae27-9b4e516c5024.png)

**Comment by compulim** [comment]
As we cannot continue to investigate without access to the environment and lacking of response from the bug author, I am closing this bug.

**Comment by pgopinadhu** [comment]
@compulim Product Team Provided the Environment access. Please check and let us know if there are any issues. Thanks!

**Comment by compulim** [comment]
No repro.

I can hear "Bot iris said I can help you with that. Please enter the sales order number to create an invoice." when the message arrive, and while navigating the transcript.

https://user-images.githubusercontent.com/1622400/224418423-40dc8dc2-58fa-464e-a2ec-43f1727af1d3.mp4
